nengo/dists.py

```diff
@@ -221,18 +221,32 @@ class UniformHypersphere(Distribution):
     surface : bool, optional (Default: False)
         Whether sample points should be distributed uniformly
         over the surface of the hyperphere (True),
-        or within the hypersphere (False).
+        or within the hypersphere (False). Defaults to False.
+    min_magnitude : Number, optional (Default: 0)
+        Lower bound on the returned vector magnitudes (i.e.
+        magnitude >= min_magnitude). Must be in the range [0, 1]; defaults
+        to 0. Values outside this range will be clipped to this range. Ignored
```

Comment:
I think all this docstring is saying is that values for ``min_magnitude`` will be clipped to [0, 1]. I agree that we should have an exception instead of clipping. This is already done by the ``low`` and ``high`` in the parameter definition.